src/lightning/pytorch/strategies/hpu_parallel.py

```diff
@@ -99,24 +99,6 @@ def setup_environment(self) -> None:
     def determine_ddp_device_ids(self) -> None:
         return None
 
-    def _pre_configure_ddp(self) -> None:
```

Comment:
This is dead code. It was missed in https://github.com/Lightning-AI/lightning/pull/16492